### PR TITLE
DOC: updating documentation 

### PR DESCRIPTION
**Pull request by YektaY**
## Description
Update to the PyDM Documentation.

Changes 
- Merged in the tutorial documentation. (https://github.com/slaclab/pydm-tutorial)
- Removed inherited members 
- Added links to the docs of inherited pyqt widgets 
- made small fixes to the auto documentation 
- updated contributors.txt file 

  
## Motivation and Context
After sourcing feedback on the current PyDM Documentation. The development team has found the current Documentation is mostly ignored by users. who find it to be too cluttered and /or missing information.

## How Has This Been Tested?
Tested locally.

### CODE REVIEW

**Review by jbellister-slac — COMMENTED** [review]
This is looking good, the cleanup of the inherited methods looks much nicer. I'll take one last pass once this is rebased. Main thing to do is just a quick find/replace on the images under the tutorials directory, one example below.

File: docs/source/tutorials/intro.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+This file is not a complete Virtual Machine dump that can be imported but instead a disk.
+
+In order to use this disk, start by creating a new virtual Machine, select Type as ``Linux`` and Version as ``Ubuntu (64-bit)``.
+Configure the amount of memory to use (something greater or equal 2048MB should do it.
+Make sure to select ``Use an existing virtual hard disk file.`` and select the extracted ``.vmdk`` file.
+
+.. figure:: /_static/new_vm.png
+    :scale: 100 %
+    :align: center
+    :alt: Create new VM
```
Comment (jbellister-slac) [inline docs/source/tutorials/intro.rst:33]: All links to `/_static/<img_file> ` should now go to `/_static/tutorials/<img_file>`

Comment (YektaY) [reply docs/source/tutorials/intro.rst]: Thanks for pointing this out, it should be resolved now

**Review by jbellister-slac — APPROVED** [review]
Thanks, this looks good to me now!

**Pull request merged**